lib/srv/uacc/uacc.h

```diff
@@ -45,6 +44,26 @@ int UACC_UTMP_FAILED_TO_SELECT_FILE = 6;
 // This decision is one of the origins of `strncpy` which has the special property that it will not write a NUL terminator if the
 // source string excluding the NUL terminator is of equal or greater length in comparison to the limit parameter.
 
+// get_absolute_with_fallback attempts to resolve the `supplied_path` path. If `supplied` is null it will
+// resolve the `fallback_path` path. The resolved path is stored in `buffer` and will at most be
+// `PATH_MAX` long including the null terminator.
+static char* get_absolute_with_fallback(char* buffer, const char* supplied_path, const char* fallback_path) {
```

Comment:
nit: I still think `get_absolute_path_with_fallback` is more readable as it immediately suggests the context of the operation.